packages/ra-core/src/controller/input/ReferenceInputController.js

```diff
@@ -172,7 +173,7 @@ export class ReferenceInputController extends Component {
         } = props;
         const { pagination, sort, filter } = this.state;
 
-        crudGetMatching(
+        crudGetMatchingDebounce(
```

Comment:
or we can simply used `lodash/debounce` here to avoid creating a new action creator and saga...

Reply:
And this happens when used inside an `ArrayInput` for example